test_boosters.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.5"
   spec.add_development_dependency "activesupport", "~> 4.0"
 
-  spec.add_development_dependency "rubocop", "~> 0.47.1"
+  spec.add_development_dependency "rubocop", "~> 0.49.0"
```

Comment:
What I'm usually seeing in gems across the globe 🌍 is that they don't "lock" gem versions in the gemspec file. Instead, they "loosen up" the gem's dependencies.

E.g. if you know that versions < 0.49.0 are bad for users and your gem, you just put ">= 0.49.0" version so people can use other versions in their projects :)

Reply:
Also, this shouldn't be done for all dependencies. It's best to read ["Pessimistic version constraint" at RubyGems docs](http://guides.rubygems.org/patterns/)

Reply:
Thanks @nikolalsvk, so this way we could cause conflicts for user if he specifies other rubocop version in his Gemfile. I'll change this to `>= 4.49.0`

Reply:
Are you sure this is relevant to gems in a user's Gemfile? Are development dependencies installed when `test-boosters` is included in some Gemfile?

Reply:
Hmm, this is development dependency therefore it won't be installed, right?

Reply:
@strika nope, this is not relevant to users using this gem since it's a development dependency. It can be important to the ones contributing 🤔

Reply:
In that case, maybe it's good to force all contributors to use the same Rubocop version. :) That way all contributors use the same style rules.

Reply:
@strika @bmarkons development dependencies can be installed with `gem install --dev test-boosters` so end-user should be safe from rubocop version blocking him in any way.

Reason for this comment is I came across some `~>` specified versions when I updated gems. Owners of those gems are not active anymore and the locked deps are blocking us from upgrading our gems.